CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 BREAKING CHANGES:
 * Helm
-  * Using the Vault integration requires Consul 1.12.0+. [[GH-1213](https://github.com/hashicorp/consul-k8s/pull/1213)]
+  * Using the Vault integration requires Consul 1.12.0+. [[GH-1213](https://github.com/hashicorp/consul-k8s/pull/1213)], [[GH-1218](https://github.com/hashicorp/consul-k8s/pull/1218)]
```

Comment:
Sorry not trying to be nitpicky here, but should we specify if this is Vault as the connect-ca, or the vault secrets backend or both?

Reply:
I think most users who are using Vault for either will be ok migrating to 1.12 which is why this was left generic.